packages/xo-web/src/common/xo/migrate-vm-modal/index.js

```diff
@@ -141,8 +141,15 @@ export default class MigrateVmModalBody extends BaseComponent {
       return
     }
 
-    const { pools, vbds, vm } = this.props
+    const { pifs, pools, vbds, vm } = this.props
     const intraPool = vm.$pool === host.$pool
+    const migrationNetwork = pools[host.$pool].otherConfig['xo:migrationNetwork']
+    const defaultMigrationNetworkId = find(
```

Comment:
The default selection of the PIF management network will change the default behavior of the Intra-pool migration.

Reply:
It's same behavior when migrate multiple VMs.